Dockerfile

```
@@ -0,0 +1,17 @@
+FROM node:latest
+
+# Set Node.js memory limit
+ENV NODE_OPTIONS="--max-old-space-size=8192"
+
+# Install cspell globally
+RUN npm install -g cspell
```

Hadolint (2.12.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)